src/test/java/io/camunda/zeebe/process/test/util/Utilities.java

```diff
@@ -152,7 +152,7 @@ public static void increaseTime(final InMemoryEngine engine, final Duration dura
     engine.increaseTime(duration);
     try {
       // we need to wait some physical time so that InMemoryEngine has a chance to fire the timers
-      Thread.sleep(50);
+      Thread.sleep(100);
```

Comment:
I know you will probably comment on this, so let me say in advance that this is temporary until I have a better solution. It's also not part of a public interface or anything, purely used for our own unit tests. 😄

Reply:
LOL 😆 

On a serious note, thanks for pointing this out. That's very useful 👍